tests/integration/conftest.py

```diff
@@ -937,17 +1255,21 @@ def assert_tests(self):
             test.assert_exists()
             test.assert_executed()
 
-    def assert_invoke_zigzag(self):
+    def assert_invoke_zigzag(self, force_clean_up=True):
```

Comment:
Implemented this for convenience when I want to re-run without cleaning up first. Turns out this pattern is pretty common.